wp-seo-main.php

```diff
@@ -414,6 +414,10 @@ function wpseo_admin_init() {
 $register_capabilities = new WPSEO_Register_Roles();
 $register_capabilities->register_hooks();
 
+// Compatibility with Members.
+$capability_members_integration = new WPSEO_Capability_Members_Integration();
```

Comment:
This class should not be instantiated at this point/time.

The plugin hooks into other plugin functionality, so `plugins_loaded` would be a logical action to hook this on.

As the functionality is only needed in the Admin area, adding it to `WPSEO_Admin` would also seem like the best place.

Reply:
> The plugin hooks into other plugin functionality, so plugins_loaded would be a logical action to hook this on.

True, at this point it wont matter since it only registers hooks. Where should I put this then?

> As the functionality is only needed in the Admin area, adding it to WPSEO_Admin would also seem like the best place.

Not always, for example, my own plugin View Admin As uses the Admin Bar in which you can manage your roles, even on the front end when logged in.
Wether Yoast would like to support that is up to you of course :). I can create my own logic to load it on the front-end when needed.